integration-libs/opf/checkout/components/opf-checkout-payments/opf-checkout-payments.component.spec.ts

```diff
@@ -172,4 +177,37 @@ describe('OpfCheckoutPaymentsComponent', () => {
 
     expect(component.selectedPaymentId).toBe(defaultSelectedPaymentOptionId);
   });
+
+  it('should render payment provider logo', () => {
+    activeConfigurationsState$.next({
+      loading: false,
+      error: false,
+      data: mockActiveConfigurations,
+    });
+
+    fixture.detectChanges();
+
+    mockActiveConfigurations.forEach((configuration) => {
+      const logoElement = el.query(
+        By.css(
+          'label[for=paymentId-' + configuration.id + ']  .cx-payment-logo'
+        )
+      );
+
+      if (configuration?.logoUrl) {
+        expect(logoElement).toBeTruthy();
+        expect(logoElement.nativeElement.attributes['class'].value).toBe(
```

Comment:
Nitpick: I think it is redundant check, since you already use it in selector `By.css('label[for=paymentId-' + configuration.id + '] .cx-payment-logo')`. So this class will be always there, or the element will be undefined at all